app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  has_many :posts
+  # Include default devise modules. Others available are:
```

Comment:
Don't leave out commented out code. Also please whitespace between each sections in classess, here between associations and device setup.